Pages/Language.cs

```diff
@@ -0,0 +1,136 @@
+﻿using NUnit.Framework.Legacy;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using QAMars.Utilities;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QAMars.Pages
+{
+    public class LanguageTest :CommonDriver
+    {
+
+        public readonly IWebDriver driver;
+        public LanguageTest(IWebDriver driver)
+        {
+            this.driver = driver;
+
+        }
+
+        public void AddLanguage(IWebDriver driver, string language, string level)
+        {
+            Wait.WaitToBeVisible(driver,"XPath", "//div[@data-tab='first']//div[text()='Add New']", 5);
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
Do I need to change any folder structure ?

Reply:
you need to look at the example and flow the same way! you have not done so